microsoft-365/compliance/email-encryption.md

```diff
@@ -61,6 +61,12 @@ Watch this video for an introduction to [Encryption in Office 365](https://www.y
 |Recommendations and example scenarios|We recommend using OME when you want to send sensitive business information to people outside your organization, whether they're consumers or other businesses. For example:  <br/>  A bank employee sending credit card statements to customers  <br/>  A doctor's office sending medical records to a patient  <br/>  An attorney sending confidential legal information to another attorney|We recommend using IRM when you want to apply usage restrictions as well as encryption. For example:  <br/>  A manager sending confidential details to her team about a new product applies the "Do Not Forward" option.  <br/>  An executive needs to share a bid proposal with another company, which includes an attachment from a partner who is using Office 365, and require both the email and the attachment to be protected.|We recommend using S/MIME when either your organization or the recipient's organization requires true peer-to-peer encryption.  <br/>  S/MIME is most commonly used in the following scenarios:  <br/>  Government agencies communicating with other government agencies  <br/>  A business communicating with a government agency|
 ||
 
+> [!NOTE]
+> In case of using both [Azure Information Protection](https://docs.microsoft.com/microsoft-365/compliance/protect-information) and email encryption to protect the data, consider following:
```

Comment:
This is confusing. AIP includes email encryption. Sensitivity labels != email encryption. I'm not sure what this is all about.

Reply:
Hello, @KCCross ,
**Sensitivity labels != email encryption** - totally agree
Sensitivity labels are applied to email by the rules.
In case you are using S/MIME encryption and sensitivity labels, you should keep in mind that the labels will not be applied, as encrypted emails are not processed by the rules. (But labels will be applied to emails signed by S/MIME)
Thank you